packages/rich-text/src/component/index.js

```diff
@@ -156,7 +156,12 @@ class RichText extends Component {
 			}
 
 			this.editableRef = node;
+
+			// This uses a native event listener because WritingFlow uses native event listener as well
+			// Event bubbling won't work properly otherwise between the two components.
+			this.editableRef.addEventListener( 'keydown', this.onKeyDown );
```

Comment:
It looks like this change broke the "splitting/merging". When you merge blocks, the caret is not put at the right position. 

I'm not certain how this works in the recent versions, I might need some help @ellatrix